src/player.cpp

```diff
@@ -10427,6 +10428,11 @@ void player::rooted()
 
 bool player::can_wield( const item &it, bool interactive ) const
 {
+    if( it.is_null() ) {
```

Comment:
Wielding a null item could simplify some code. For example, "wield the best weapon you have" could start with a null item (fists), compare all items and try to wield the one with best values.
While it would need to be special cased in `wield` (we always want to wield our "personal" null item - `player::ret_null`), this special case might not be needed here.